gui/default/assets/css/overrides.css

```diff
@@ -569,10 +565,7 @@ html[lang|="ko"] i {
     user-select: all;
 }
 
-#login-options .panel-body {
-  padding: 1em;
-}
-
+/* Align checkbox with button in login forms */
```

Comment:
Ah, I see.  Maybe there's a better way to align them by centering vertically in some common container?  The Sharing tab when editing a folder / device wraps a checkbox and a text entry into a common `<div class="form-group">`, maybe that would be an easier solution here as well?

Reply:
Nope, these elements are already wrapped with `.form-group`, and the only thing that class does is `margin-bottom: 15px;`.

The correct solution to this would be to use a [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox) with `align-items: baseline` to align the text baselines of the checkbox label and the button. But flexbox is somewhat new, and I've seen some concerns about compatibility with older browsers, so I didn't reach for that solution for now. With your blessing I would be happy to do that, though.

Reply:
Shouldn't this be really simple? https://stackoverflow.com/a/7311753

Reply:
Unfortunately no...

`display: inline-block` and `vertical-align: middle` works for getting them aligned vertically, but breaks the horizontal right-alignment of the button. The right-alignment is currently achieved using the `col-md-9` and `col-md-3` classes, which set the `float` property which forces `display:` to `block`. I tried a couple other things but didn't get both vertical alignment and button-right-alignment to work simultaneously.

Actually, flexbox has been [widely supported since 2015](https://caniuse.com/flexbox), so maybe we should just use that after all. In browsers that support it, it'll look like this:

![image](https://github.com/user-attachments/assets/671e416c-c71b-4be0-834a-0d15d59e96cc)

And in browsers that don't support it, it'll be ugly but still functional:

![image](https://github.com/user-attachments/assets/bc16d923-24b5-4309-ad0a-bab341090799)

Does that seem acceptable?

Reply:
Actually I think having the button on the right side is more important than the vertical alignment.

Taking a step back, why don't we just put the button on a row of its own, as in all other dialogs.  It's just the utter desire of utilizing the empty space left of the button for the checkbox that gets us into this trouble.  Everywhere else, the action buttons are alone on their row and we never have these problems.  Let's try that and see what ideas others come up with during review of the whole PR.  Acceptable?

Reply:
Ok, that sounds fair.